bugsnag-plugin-android-ndk/src/main/java/com/bugsnag/android/ndk/NativeBridge.kt

```diff
@@ -172,9 +170,7 @@ class NativeBridge : StateObserver {
                     arg.autoDetectNdkCrashes,
                     Build.VERSION.SDK_INT,
                     is32bit,
-                    makeSafe(arg.appVersion ?: ""),
-                    makeSafe(arg.buildUuid ?: ""),
-                    makeSafe(arg.releaseStage ?: "")
```

Comment:
Are these definitely not required? IIRC there is a ticket in the backlog that should address whether these should be populated by native events

Reply:
They're not actually on the JNI signatures (in C) at the moment, so we definitely shouldn't be passing them down.